pkg/sqlite/load.go

```diff
@@ -20,7 +20,7 @@ type SQLLoader struct {
 
 var _ registry.Load = &SQLLoader{}
 
-func NewSQLLiteLoader(outFilename string) (*SQLLoader, error) {
+func NewSQLLiteLoader(outFilename, migrationsPath string) (*SQLLoader, error) {
```

Comment:
Did I hallucinate a change to this API? I thought this became variadic with config options?

Reply:
you did not, i rebased poorly :) latest should contain the right bits.